uvicorn/config.py

```diff
@@ -471,7 +473,9 @@ def load(self) -> None:
         else:
             self.ws_protocol_class = self.ws
 
-        self.lifespan_class = import_from_string(LIFESPAN[self.lifespan])
+        self.lifespan_class: Type[Union[LifespanOn, LifespanOff]] = import_from_string(
+            LIFESPAN[self.lifespan]
+        )
```

Comment:
This is just nice to have typing that avoids adding typing downstream in `server.py`

Reply:
We can't use the `Type[Lifespan]` here?

btw, the `LifespanOn` and `LifespanOff` are in the `TYPE_CHECKING`, but here they are not string annotated.

Reply:
Changed/fixed